step-templates/Jenkins-Queue-Job

```diff
@@ -0,0 +1,71 @@
+    "Octopus.Action.Script.ScriptSource": "Inline",
+    "Octopus.Action.RunOnServer": "false",
+    "Octopus.Action.Script.ScriptBody": "$JenkinsServer = \"http://jenkins:8080/\"\r\n$jobURL = $JenkinsServer + $OctopusParameters['JobUrl']\r\n$failBuild = $OctopusParameters['FailBuild']\r\n$jobTimeout = $OctopusParameters['JobTimeout']\r\n$buildParam = $OctopusParameters['BuildParam']\r\n$buildResult = \"/lastBuild/api/json?tree=result,number,building\"\r\n\r\nWrite-Host \"job url: \" $jobURL$buildParam \r\n$response = Invoke-WebRequest -Uri $jobURL$buildParam  -Method Post\r\n\r\nif ($failBuild -eq \"True\")\r\n{\r\n   $isBuilding=\"True\"\r\n   $i = 0\r\n   Write-Host \"Estimate Job Duration: \" $jobTimeout\r\n   while ($isBuilding -eq \"True\" -and $i -lt $jobTimeout)\r\n   {       \r\n       $i +=20\r\n       Write-Host \"waiting $i secs for build to complete\"\r\n       Start-Sleep -s 20\r\n       $retyJobStatus = Invoke-RestMethod -Uri $jobURL$buildResult -Method Get\r\n       $isBuilding=$retyJobStatus[0].building\r\n       $result = $retyJobStatus[0].result\r\n       $buildNumber = $retyJobStatus[0].number\r\n       Write-Host \"Retry Job Status: \" $result \" BuildNumber: \"  $buildNumber  \" IsBuilding: \"  $isBuilding \r\n   }\r\n   \r\n   if ($result -ne \"SUCCESS\")\r\n   {\r\n       echo \"BUILD FAILURE: build is unsuccessful or status could not be obtained.\"\r\n       exit 1\r\n   }\r\n}",
+    "Octopus.Action.Script.ScriptFileName": null,
+    "Octopus.Action.Package.FeedId": null,
+    "Octopus.Action.Package.PackageId": null
+  },
+  "Parameters": [
+    {
+      "Id": "b8337514-3989-4b33-930c-b5ebde5b4be0",
+      "Name": "JobUrl",
+      "Label": "Job Url",
+      "HelpText": "e.g. job/projectname/build",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      },
+      "Links": {}
+    },
+    {
+      "Id": "a52f7318-6f45-4e9f-b825-b3ae767608f8",
+      "Name": "FailBuild",
+      "Label": "Fail Build",
+      "HelpText": "Should this fail the deployment?",
+      "DefaultValue": "No",
+      "DisplaySettings": {
+        "Octopus.ControlType": "Checkbox"
+      },
+      "Links": {}
+    },
+    {
+      "Id": "299a4df6-dd1c-4e38-9d9e-d64c69c9cc65",
+      "Name": "JobTimeout",
+      "Label": "Timeout Duration(secs)",
+      "HelpText": "e.g. 60. Specify in secs how long to check for the job status before timing out.",
+      "DefaultValue": "0",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      },
+      "Links": {}
+    },
+    {
+      "Id": "36b50c4e-e086-4e6a-ac91-311cf5dc3bef",
+      "Name": "BuildParam",
+      "Label": "Build Param",
+      "HelpText": "e.g. ?Param=Value or ?delay=10sec",
+      "DefaultValue": "/build",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      },
+      "Links": {}
+    }
+  ],
+  "LastModifiedOn": 2017-08-11T15:50:26.518Z",
+  "LastModifiedBy": "George Nelson",
```

Comment:
If you update this to your :octocat: handle, it will be nicely linked on the Library website